articles/users/search/v2/index.md

```diff
@@ -6,6 +6,10 @@ toc: true
 
 # User Search
 
+::: version-warning
+This document covers a previous version of user search. We recommend you use <a href="/users/search/v3">user search v3</a>.
```

Comment:
@erjohnson For future reference, you don't have to use HTML within the UI elements, you can use Markdown and the result is the same. I'm going to fix these two for you and merge.